src/clj/ataru/hakija/hakija_form_service.clj

```diff
@@ -53,33 +53,57 @@
           :public-config
           (get :attachment-modify-grace-period-days 14))))
 
+(defn- get-hakukohteiden-hakuajat-for-oids [hakukohde-oids hakukohteet]
```

Comment:
I'd try to clarify what this function does and why, difficult name!

Reply:
Changed to find-existing-hakuajat-for-set-of-hakukohde-oids